app/src/main/java/org/wikipedia/page/PageFragmentLoadState.kt

```diff
@@ -38,6 +38,8 @@ class PageFragmentLoadState(private var model: PageViewModel,
                             private var leadImagesHandler: LeadImagesHandler,
                             private var currentTab: Tab) {
 
+    private var isLoading = false
```

Comment:
I'm afraid this also adds bugs, including (but not limited to) inability to recover if an exception is thrown while loading a page.
Are we sure this is necessary?